md/architecture.md

```diff
@@ -24,6 +24,10 @@ The "Operations Management" Living Application is a mobile first -usable on a de
  
 The backend server polls data, creates process mining models, and serves the REST APIs.
 
+:::info
+ICI Server is secured by https and a security token to allow only requests from Bonita server.
```

Comment:
- I think this is the time to start renaming ICI in BICI wherever possible: at least not add any new "ICI" occurrence, at most change all occurrences of the page. It's ok not to be consistent for a while, before other pages will become compliant.
- Is that the location to explain that if the customer is not "equipped" for https, and for BICI evaluation only, we provide a self-signed certificate, so as not to scared anyone?
- English "only allow"
- BICI server, with lower 's', as this is not a markatable element, but "just" one piece of the architecture. (We capitalize Bonita Studio, UI Designer, Bonita Engine, as they are marketable elements of the solution).

Reply:
Since secured token is not intend to be provided by patty, it's a different approach that https certificate configuration, which is detailed in installation guide
we can discuss this point

Reply:
ok. Good. Thanks for your answer.